### PR TITLE
Add dropdown buttons to main page and tabs to projects page to allow user to control content displayed.

### PR DESCRIPTION
**Pull request by nikhitashanker**
Currently, all content is displayed in all pages. For example, on the projects page, all projects are shown one after another to the user such that scrolling is necessary. In this change, add dropdown buttons to the main page and tabs to the project page. When the dropdown buttons are clicked on the main page, corresponding information about me is shown. When tab buttons are clicked on the project page, the corresponding project information is displayed.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +35,33 @@ function addRandomFact() {
   const randomFactContainer = document.getElementById('random-fact-container');
   randomFactContainer.innerText = fact;
 }
+
+/*
+  * Expands content when content title button is clicked.
+*/
+function toggleContentVisibility(){
```
Comment (hlz03) [inline portfolio/src/main/webapp/script.js:42]: nit: fix comment styles.

Comment (liangqigong) [reply portfolio/src/main/webapp/script.js:42]: I think you may want to find an auto formatter tool so avoid this kind of issue.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +35,33 @@ function addRandomFact() {
...
+  // Toggle the visibility of the content.
+  if (content.style.display === "block") {
+      content.style.display = "none";
+  } else {
+      content.style.display = "block";
+  }
+}
+
+/*
+  * Show project content that corresponds to selected tab.
```
Comment (liangqigong) [inline portfolio/src/main/webapp/script.js:54]: Ditto.

File: portfolio/src/main/webapp/projects.html [hunk 3]
```diff
@@ -9,15 +9,23 @@
...
-      <center><img src="images/5MinutesADay.png" alt="5 Minutes A Day" width="250" height="400"></center>
-      <h4>UC Brainstorm</h4>
-      <p>UC Brainstorm is a web application that helps users brainstorm answers to the UC Application's short answer questions.</p>
-      <center><img src="images/UCBrainstorm.png" alt="UC Brainstorm" width="500" height="250"></center>
+      <button class="select-tab" onclick="showContentForTab('swe-dev-portal')">SWE Dev Portal</button>
+      <button class="select-tab" onclick="showContentForTab('five-minutes-a-day')">5 Minutes A Day</button>
+      <button class="select-tab" onclick="showContentForTab('uc-brainstorm')">UC Brainstorm</button>
+      <div id="swe-dev-portal" class="tab-content">
+        <h4>SWE Dev Portal</h4>
+        <p>I am currently the technical lead and a developer for the Society of Women Engineers development team at UCLA. We are building a member <a href=https://github.com/swe-ucla/Event-Portal>portal</a> that will allow users to track the events they have attended and register for Evening With Industry, a yearly event.</p>
```
Comment (liangqigong) [inline portfolio/src/main/webapp/projects.html:17]: I know there is no hard line limit, but it seems a little bit too long for me. Did you tried any html auto formatting tool? Did it break the line?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (nikhitashanker) [reply portfolio/src/main/webapp/script.js:42]: I have not tried any auto-formatter tool. Prettier.io seems to have different format than the style guide. Is there any tool you recommend to use to auto-format according to Google style guide?

File: portfolio/src/main/webapp/projects.html (hunk 3 above)
Comment (nikhitashanker) [reply portfolio/src/main/webapp/projects.html]: I agree, I have added line wrapping. I didn't try any auto formatting tool, but it was out of view on the github and my IDE. Done.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (nikhitashanker) [reply portfolio/src/main/webapp/script.js]: Done.

**Pull request merged**